requirements.txt

```diff
@@ -1,4 +1,4 @@
-git+https://github.com/adsabs/ADSPipelineUtils.git@master
+git+https://github.com/adsabs/ADSMicroserviceUtils.git@master
```

Comment:
this repo doesn't exist yet; but i'll create it - you had the right idea to reuse the library for logging/config

the problem is this module brings in lot of stuff that is only useful for celery workers (pipelines)

Reply:
this is good thank you, and easy to replace utils.